src/Helpers/PageHelpers/PageHelper.cs

```diff
@@ -714,5 +714,41 @@ public void CheckForBookingElement(List<Page> pages)
                     throw new ApplicationException("PageHelper:CheckForBookingElement, Booking element requires customer firstname/lastname elements for reservation");
             }
         }
+
+        public void CheckAbsoluteDateValidations(List<Page> pages)
+        {
+            var elements = pages.SelectMany(element => element.ValidatableElements);
+            
+            elements.Where(element => !string.IsNullOrEmpty(element.Properties.IsDateAfterAbsolute))
+                .ToList()
+                .ForEach(element => { 
+                                        if(!DateTime.TryParse(element.Properties.IsDateAfterAbsolute, out DateTime outputDate)){ throw new ApplicationException($"PageHelper:CheckDateValidations, IsDateAfterAbsolute validation, {element.Properties.QuestionId} does not provide a valid comparison date"); }
+                                    });
+
+            elements.Where(element => !string.IsNullOrEmpty(element.Properties.IsDateBeforeAbsolute))
+                .ToList()
+                .ForEach(element => { 
+                                        if(!DateTime.TryParse(element.Properties.IsDateBeforeAbsolute, out DateTime outputDate)){ throw new ApplicationException($"PageHelper:CheckDateValidations, IsDateBeforeAbsolute validation, {element.Properties.QuestionId} does not provide a valid comparison date"); }
+                                    });
+            
+            
+        }
```

Comment:
The contents of the If statements should be on a new line and the indentation needs to be fixes. Remove blank lines after foreach and before the end of the method. The same needs to be fixed below.